syntaxes/highlight-sql-string.json

```diff
@@ -5,19 +5,17 @@
   "injectionSelector": "L:string.quoted.multi.python, L:meta.fstring.python - (comment.line.number-sign.python, punctuation.definition.comment.python)",
   "patterns": [
     {
-      "begin": "( *--sql| *--beginsql| *--begin-sql)",
```

Comment:
I do not recommend to remove the current identifier. It would break all implementation. Add new ones but do not remove the current ones.

Reply:
Sure, thanks for getting to this :)

Reply:
Ah, you are not maintainer haha. You're probably right but ill wait to hear from repo owner/member if they are still alive...